syntax = "proto3";

message LocationMessage {
    string person_id = 1;
    string latitude = 2;
    string longitude = 3;
}   

message LocationMessageRequest {
    string id = 1;
}

message LocationMessageList {
  repeated LocationMessage locations = 1;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
    rpc Get(LocationMessageRequest) returns (LocationMessage);
}

